README.md

```diff
@@ -1 +1,191 @@
+  
+  git clone git@github.com:jcmunav63/metrics_app_mm_capstone_m3.git
+```
+
+### Prerequisites <a name="prerequisites"></a>
+
+In order to run this project you need the following tools:
+- A modern web browser (Chrome, Firefox, Safari, Edge).
+- An IDE (e.g.: Vscode, Sublime Text,...).
+- A terminal (Git Bash).
+- Create a React project, including Webpack project bundler.
+
+
+### Install <a name="install"></a>
+
+- Install this project with linters and Webpack bundler:
+
+```sh
+  cd metrics_app_mm_capstone_m3
+
+  npx create-react-app .  (inside of the root directory)
+
+  npm init -y
+
+  npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+  npm install --save-dev eslint@7.x eslint-config-airbnb@18.x eslint-plugin-import@2.x eslint-plugin-jsx-a11y@6.x eslint-plugin-react@7.x eslint-plugin-react-hooks@4.x @babel/eslint-parser@7.x @babel/core@7.x  @babel/plugin-syntax-jsx@7.x  @babel/preset-react@7.x @babel/preset-react@7.x
+```
+
+### Usage <a name="usage"></a>
+
+To run the project, execute the following command:
+
+```sh
+  Open the project in Webpack dev server or Open the index.html file, from dist/ folder, on your browser.
+```
+### Build the project with Webpack and run a dev server
+```
+  Run the following commands in the terminal:
+
+  npm run build
+  
+  npm start
+```
+
+### Run Linters tests <a name="run-tests"></a>
+
+To run linters checking, run the following command:
+
+```sh
+
+  npx stylelint "**/*.{css,scss}" (--fix option)
+
+  npx eslint "**/*.{js,jsx}" (--fix option)
+```
+
+### Deployment <a name="triangular_flag_on_post-deployment"></a>
+
+🚀 Project deployment.
+
+Soon to be deployed...
+- https://
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Peter Mwovi**
+
+- GitHub: [@githubhandle](https://github.com/peter-mwovi/)
+- Twitter: [@twitterhandle](https://twitter.com/mwovi_peter)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/peter-mwovi-57141a179/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+The Metrics App will include the following features:
+- **No additional features.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+All contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐ Show your support <a name="support"></a>
+
+If you like this project, please support me with a like or a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Thanks to my learning partners and the staff at Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] The README should contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
- [ ] As requested by the previous reviewer, Kindly add the acknowledgments of the original designer to the `Acknowledgements` section part of the readme file [here](https://github.com/peter-mwovi/metrics_webapp/pull/4#discussion_r1367795095)

![image](https://github.com/peter-mwovi/metrics_webapp/assets/81932478/0a63bab0-cf18-4879-abfb-52eb5c059850)